src/apps/service/apps/de/pipeline_edit.clj

```diff
@@ -1,22 +1,21 @@
 (ns apps.service.apps.de.pipeline-edit
-  (:use [korma.core :exclude [update]]
-        [korma.db :only [transaction]]
-        [kameleon.core]
-        [kameleon.entities]
-        [kameleon.uuids :only [uuidify]]
-        [medley.core :only [find-first]]
-        [apps.persistence.app-metadata :only [add-app
-                                                    add-mapping
-                                                    add-step
-                                                    add-task
-                                                    get-app
-                                                    remove-app-steps
-                                                    update-app]]
-        [apps.util.conversions :only [remove-nil-vals]]
+  (:use [apps.persistence.app-metadata :only [add-app
+                                              add-mapping
+                                              add-step
+                                              add-task
+                                              get-app
+                                              remove-app-steps
+                                              update-app]]
+        [apps.persistence.entities :only [app_steps input_mapping]]
+        [apps.service.apps.de.edit :only [add-app-to-user-dev-category app-copy-name]]
         [apps.service.apps.de.validation :only [verify-app-editable]]
+        [apps.util.conversions :only [remove-nil-vals]]
```

Comment:
Yep. I think I'll keep it out of this PR (since this PR just rearranged the `:require` deps and didn't introduce this function to this namespace or service).

But it's a good idea for another commit/PR after this PR.